src/js/angular/security/controllers.js

```diff
@@ -104,6 +104,37 @@ const parseAuthorities = function (authorities) {
     };
 };
 
+const compactAuthorities = function (authorities) {
+    const writeAllRepos = authorities.find(authority => authority === `${WRITE_REPO_PREFIX}*`);
```

Comment:
`writeAllRepos` is not a good name, better name would be something like `writeAllRole` or `writeAllAuthority`.